src/generators/static/README.md.hbs

```diff
@@ -0,0 +1,101 @@
+# {{ clientDescriptiveName }} library for JavaScript
+
+This package contains an isomorphic SDK (runs both in node.js and in browsers) for {{ clientDescriptiveName }}.
+
+{{ description }}
+
+{{#if packageSourceURL}}
+[Source code]({{ packageSourceURL }}) |
+{{/if}}
+{{#if packageNPMURL}}
+[Package (NPM)]({{ packageNPMURL }}) |
+{{/if}}
+{{#if apirefURL}}
+[API reference documentation]({{ apirefURL }}) |
+{{/if}}
+{{#if samplesURL}}
+[Samples]({{samplesURL}})
+{{/if}}
+
+## Getting started
+
+### Currently supported environments
+
+- Node.js version 8.x.x or higher
+- Browser JavaScript
+
+{{#if azure}}
+### Prerequisites
+
+- An [Azure subscription][azure_sub].
+{{/if}}
+
+### Install the `{{ clientPackageName }}` package
+
+Install the {{ clientDescriptiveName }} library for JavaScript with `npm`:
+
+```bash
+npm install {{ clientPackageName }}
+```
+
+### Create and authenticate a `{{ clientClassName}}`
+
+To create a client object to access the {{ serviceName }} API, you will need the `endpoint` of your {{ serviceName }} resource and a `credential`.{{#if azure}} The {{ clientDescriptiveName }} can use Azure Active Directory credentials to authenticate.
+You can find the endpoint for your {{ serviceName }} resource in the [Azure Portal][azure_portal].
+
+#### Using an Azure Active Directory Credential
```

Comment:
Shouldnt this section be conditional based on whether or not the service supports AAD?

Reply:
Made it behind the ``add-credentials` autorest argument